pkgs/os-specific/linux/firmware/b43-firmware-cutter/default.nix

```diff
@@ -11,7 +11,11 @@ stdenv.mkDerivation rec {
 
   patches = [ ./no-root-install.patch ];
 
-  makeFlags = [ "PREFIX=$(out)" ];
+  makeFlags = [
+    "PREFIX=$(out)"
+  ] ++ lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [
+    "CC=${stdenv.hostPlatform.config}-cc"
```

Comment:
you can use `"CC=${stdenv.cc.targetPrefix}cc"` unconditionally instead.

Reply:
Updated to use targetPrefix! Thanks.